packages/api/tests/task.test.js

```diff
@@ -7,10 +7,7 @@ jest.mock('jsdom');
 jest.mock('../src/middleware/acl/checkJwt');
 const mocks = require('./mock.factories');
 const { tableCleanup } = require('./testEnvironment');
-const moment = require('moment');
-let faker = require('faker');
-const { test } = require('../knexfile');
```

Comment:
@MwayaB I removed several unneeded `require`s in this area. The one for `test` was actually causing a problem I think. I was getting "test is not a function" when I first ran the tests. I'm not sure what `test` in the knexfile is, but it's not the `test` we need here. 
Without the `require`, eslint does complain that it does not know what `test` is. That semi-problem is addressed at the bottom of this file, where I have commented on the fix.